ops/terraform/variables.tf

```diff
@@ -1,12 +1,16 @@
 variable "client_id" {}
 variable "name" {}
+variable "postgres_server_id" {}
```

Comment:
do we need this variable here? cause if we do we have to set it in the tfvars file